types/discord.ts

```diff
@@ -85,22 +85,42 @@ export interface DiscordConnection {
   /** The username of the connection account */
   name: string;
   /** The service of the connection (twitch, youtube) */
-  type: string;
+  type: DiscordConnectionServices;
   /** Whether the connection is revoked */
   revoked?: boolean;
   /** Whether the connection is verified */
   verified: boolean;
   /** Whether friend sync is enabled for this connection */
-  friendSync: boolean;
+  friend_sync: boolean;
   /** Whether activities related to this connection will be shown in presence updates */
-  showActivity: boolean;
+  show_activity: boolean;
   /** Visibility of this connection */
   visibility: VisibilityTypes;
 
   /** An array of partial server integrations */
   integrations?: DiscordIntegration[];
 }
 
+/** https://discord.com/developers/docs/resources/user#connection-object-services */
+export enum DiscordConnectionServices {
```

Comment:
Why is this an enum? Generally, an enum should be used for numbers imo, since numbers are harder to read and know it's meaning. In this case it is a string `ebay` is clear where as type `0` for channel is not clear that it is a text channel.

Reply:
This should be a `export type DiscordConnectionServices = "ebay" | "x" | "y"......;`